packages/accounts-controller/src/AccountsController.ts

```diff
@@ -1218,7 +1220,7 @@ export class AccountsController extends BaseController<
     // Handle account change when multichain network is changed
     this.messagingSystem.subscribe(
       'MultichainNetworkController:networkDidChange',
-      this.#handleMultichainNetworkChange,
+      this.#handleOnMultichainNetworkChange,
```

Comment:
This is not entirely valid here too. Referencing the callback like this would not be bind automatically to `this` (at least in classic javascript), IDK if this applies while transpilling from ts to js though...

Looks like it's not really valid anyway, I've tested with this: https://www.typescriptlang.org/play/?#code/GYVwdgxgLglg9mABAZwIYE8AUwBcjMCUiAvAHwpQBOMYA5kQN4BQiriECycANgKYB03OLWyECAbiYBfJkwjdUyZIgBicOACNUlRMzaJKvVABME3dIgDEXALa8oACxq08yKs9n6OYN5RDQ4Skxbeyc6V3c6RhZ9NkcYZH5rODt4uhIUFNDnSVi2AHp8lAxMeMTLCFRubi0IAGsJRELEAElIQMNoABp2VBBkXjwAIgBhVDAwOCgDI2NEAAdqADdUKF5EOxsNXh1gShTEccRNACteaEQAdwc4AfYFJURjGDnJ6eNzhUNEGCghmLyaCwZSSlWqtTq-A0NGMpScyAIEgBiBkyIqVRqqHqhF0yNYhigIEoSBByVSYVouTYMlR3jciGAGTAvEuqnUWiCAHJgOztJyJEA

To match the style of this controller, I would use an arrow function here:
```suggestion
      (networkId) => this.#handleOnMultichainNetworkChange(networkId),
```

This way we don't need to `.bind` explicitly (and this matches the overall style of the controller)

Reply:
The binding worked since the function was defined with an arrow function. But I see your point with the pattern, updated to match other styles.